security/rbac/docs/index.rst

```diff
@@ -0,0 +1,460 @@
+.. _rbac_demo:
+
+Overview
+========
+
+This demo showcases the |rbac-long| functionality in |cp|.
+It is mostly for reference to see a workflow using the new |rbac| feature across the services in |cp|.
+
+There are two ways to run the demo.
+
+-  :ref:`Local install <rbac_demo_local>`
+-  :ref:`Docker <rbac_demo_docker>`
+
+
+Additional Reading
+------------------
+
+-  `RBAC for Kafka Connect whitepaper <https://www.confluent.io/resources/rbac-for-kafka-connect>`__
+-  `RBAC documentation <https://docs.confluent.io/current/security/rbac/index.html>`__
+
+
+.. _rbac_demo_local:
+
+Local install
```

Comment:
suggest making this title more meaningful, e.g. `Demo 1: Install RBAC locally`

Reply:
Ack